build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e 	# Exit immediately upon failure
+set -o pipefail  # Carry failures over pipes
+
+repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+docker_repo="microsoft/aspnetcore"
```

Comment:
We need to also account for files that are on the `microsoft/aspnetcore-build` repo instead.